Fix build on macOS High Sierra and later
https://sourceforge.net/p/optipng/bugs/68/
https://sourceforge.net/p/optipng/patches/9/
osys.c:515:25: error: no member named 'st_atim' in 'struct stat'
osys.c:516:25: error: no member named 'st_mtim' in 'struct stat'
--- src/optipng/osys.c.orig	2014-10-04 22:51:00.000000000 -0500
+++ src/optipng/osys.c	2017-12-09 11:38:10.000000000 -0600
@@ -512,8 +512,13 @@
     {
         struct timespec times[2];
 
+#ifdef OSYS_DARWIN
+        times[0] = sbuf.st_atimespec;
+        times[1] = sbuf.st_mtimespec;
+#else
         times[0] = sbuf.st_atim;
         times[1] = sbuf.st_mtim;
+#endif
         if (utimensat(AT_FDCWD, dest_path, times, 0) != 0)
             result = -1;
     }
